### PR TITLE
RicA pull request

### PR DESCRIPTION
**Pull request by cloud-byte**
Added getDescription method

### CODE REVIEW

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearchResult.java [hunk 1]
```diff
@@ -65,5 +62,15 @@ public void clickSellerFilter(WebDriver driver,int indexElement) {
...
+	public String  getDescription() {
+		String texto = "";
+		try { //no todos los articulos tienen el mismo id en el div de descripcion
+			texto = driver.findElement(By.xpath("//div[@id='productDescription_feature_div']//div[@id='productDescription_feature_div']")).getText();
+		} catch (Exception e) {
+			texto = driver.findElement(By.xpath("//div[@id='productDescription']")).getText();
+		}
+		return texto;
+
 	}
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSearchResult.java]: This must be compressed

File: TestingPro/src/com/deloitte/test/amazon/tests/Test2Amazon.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com.deloitte.test.amazon.configuracion.Configuracion;
+import com.deloitte.test.amazon.page.home.AmazonHome;
+
+import java.util.logging.Logger;
+public class Test2Amazon {
+	//static AmazonHome ama = new AmazonHome(null);
+	private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+	public static void main (String [] args) throws InterruptedException {	
+		
+		System.out.println("Starting testcase 1");
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/tests/Test2Amazon.java]: Use Log class instance of System class

**Review by navi911407 — CHANGES_REQUESTED** [review]
Please make changes requested

**Review by cloud-byte — COMMENTED** [review]
Changes made

**Pull request merged**